features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -217,7 +217,11 @@ export function SidebarNavigation() {
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              onClick={() =>
```

Comment:
The slightly simpler option would be to use the `MenuItemLink` component instead of the button here. Then you can simply use the `href` prop with `mailto:...`. This would make the test simpler as well since you could simply check the value of the `href`